django_app/report_a_suspected_breach/forms.py

```diff
@@ -100,6 +104,20 @@ def clean_email_verification_code(self) -> str:
 
         return email_verification_code
 
+    def __init__(self, *args: object, **kwargs: object) -> None:
+        super().__init__(*args, **kwargs)
+        self.request = kwargs.pop("request") if "request" in kwargs else None
+        request_verify_code = reverse_lazy("report_a_suspected_breach:request_verify_code")
+        self.helper["email_verification_code"].wrap(
+            Field,
+            HTML(
+                f"""<p class=govuk-body>
+                <a class=govuk-link href={request_verify_code}>
+                Not received an email or code not working?
+                </a></p>"""
```

Comment:
can you please store this code as an HTML file and then call `render` to get the string, just makes it cleaner I think and avoids writing HTML in python

```suggestion
            HTML(render("report_a_suspected_breach/form_steps/partials/not_received_code_help_text"))
```